clients/client-python/build.gradle.kts

```diff
@@ -21,11 +23,52 @@ fun deleteCacheDir(targetDir: String) {
   }
 }
 
+
+fun waitForServerIsReady(host: String = "http://localhost", port: Int = 8090, timeout: Long = 60000) {
```

Comment:
How about change 60s  to 30s?  if Gravitino server failed to start, 60 seconds seems too long

Reply:
It's okay for me.